docker/docker-dev-direct-single-tenant/datasources/mt-graphite-api

```diff
@@ -1,7 +0,0 @@
-{
-  "name":"mt-graphite-api",
-  "type":"graphite",
-  "url":"http://localhost:8080",
-  "access":"direct",
-  "isDefault":true
-}
```

Comment:
i removed docker/docker-dev-direct-single-tenant completely, as it is no different to docker-dev.
All docker envs that use direct carbon input are configured as single-tenant (as all metrics ingested use orgId=1).  the cluster envs, use multiTenant metrictank, but have graphite configured in singleTenant mode so it sends the x-org-id:1 header, preventing the need for running tsdb-gw.